openmdao/core/system.py

```diff
@@ -191,14 +210,20 @@ def _setup_paths(self, parent_path):
         parent_path : str
             The pathname of the parent `System`, which is to be prepended to the
             name of this child `System`.
+
+        probdata : `_ProbData`
+            Problem level data container.
         """
-        self._fd_params = None
+        self._reset()
 
         if parent_path:
             self.pathname = '.'.join((parent_path, self.name))
         else:
             self.pathname = self.name
 
+        self._sysdata = _SysData(self.pathname)
+        self._probdata = probdata
+
```

Comment:
if we're 'reset'ing and initializing sysdata and probdata in this function then the name "setup_paths" seems inadeqaute to describe whats it's doing now